schemas/research-technology-readiness-level.jsonld

```diff
@@ -0,0 +1,128 @@
+{
+    "@context": {
+        "trl": "https://w3id.org/research-technology-readiness-level#",
+        "skos": "http://www.w3.org/2004/02/skos/core#",
+        "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+        "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+        "prefLabel": { "@id": "skos:prefLabel" },
+        "Concept": { "@id": "skos:Concept" },
+        "definition": { "@id": "skos:definition" },
+        "notation": { "@id": "skos:notation" },
+        "inScheme": { "@id": "skos:inScheme" },
+        "Property": { "@id": "rdf:Property" },
+        "label": { "@id": "rdfs:label" },
+        "comment": { "@id": "rdfs:comment" },
+        "Property": { "@id": "rdf:Property" },
+        "OrderedCollection": { "@id": "skos:OrderedCollection" },
+        "ConceptScheme": { "@id": "skos:ConceptScheme" },
+        "memberList": { "@id": "skos:memberList" }
+    },
+    "@graph": [
+        {
+            "@id": "trl:Levels",
+            "@type": "OrderedCollection",
```

Comment:
You defined the properties in the `@context`, which is completely fine. However, I prefer to keep the prefix in the `@graph` so for each property it’s immediately clear which vocabulary it belongs to:

```suggestion
            "@type": "skos:OrderedCollection",
```

(etc.)

Reply:
Okay, fair enough yes, addressed now in 222e76773c47ce811b891563f4a9fa10c416a29a